src/Engine/Component/Mesh.hpp

```diff
@@ -32,14 +32,14 @@ namespace Component {
 
             /// Get mesh selection.
             /**
-            * @return Bool Is this selected or not.
-            */
+             * @return Bool Is this selected or not.
```

Comment:
The type of the return value shouldn't be documented. Doxygen already extracts that.